BLL/ReviewService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace BLL
+{
+    public class ReviewService
+    {
+        public bool EditReview(Drink drink, int score, string review)
+        {
+            if (!drink.isReviewed)
+            {
+                return false;
+            }
+
```

Comment:
It is not necessary to check here if drink is reviewed, because switch case instruction from DisplayProductsDetailsOptions does that.